.github/workflows/rust.yml

```diff
@@ -33,6 +33,25 @@ jobs:
         run: |
           cargo test --verbose
 
+  linux-cross-compile:
```

Comment:
Should this be a required job for a successful run?

Reply:
It should.